akka-docs/src/main/paradox/cluster-client.md

```diff
@@ -1,14 +1,19 @@
 # Classic Cluster Client
 
+@@@ warning
+
+Cluster Client is deprecated in favor of using [Akka gRPC](https://doc.akka.io/docs/akka-grpc/current/index.html). 
+It is not advised to build new applications with Cluster Client, and existing users @ref[should migrate](#migration-to-akka-grpc). 
+
+@@@
+
 @@@ note
```

Comment:
After this and https://github.com/akka/akka/pull/27617 merge I'll swing back here and update this below to the new includes.